data/sound-strings/en.json/localize.json

```diff
@@ -1,4 +1,5 @@
 {
+"speed_camera": "The road camera is on the way.",
```

Comment:
I think the correct way is to edit data/sound.txt and then run python/tts_languages.py

Reply:
Correction: the script to run is tools/unix/generate_localizations.sh

Reply:
done